.github/workflows/notify-triager.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   Notify-triagers:
+    if: ! contains(github.event.pull_request.head.sha, 'Merge branch')
```

Comment:
This line is for skipping the workflow when the PR is updated. Am I correct?

If that's the case, `.sha` would give the unique commit hash, and not the actual message that contains `Merge branch`.

ref: https://github.com/orgs/community/discussions/28474#discussioncomment-3334578